### PR TITLE
SOC-1469 | Iterate over replies only if they exist

### PR DESCRIPTION
**Pull request by bkoval**
@armonr @rafalkalinski @kvas-damian https://wikia-inc.atlassian.net/browse/SOC-1469

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Comment by armonr** [comment]
:+1:

File: front/scripts/main/models/DiscussionPostModel.ts [hunk 1]
```diff
@@ -78,19 +78,19 @@ App.DiscussionPostModel.reopenClass({
...
-						if (reply.hasOwnProperty('createdBy')) {
-							author = reply.createdBy;
-							author.url = M.buildUrl({
-								namespace: 'User',
-								title: author.name
-							});
-							contributors.push(author);
-						}
-					});
+						replies.forEach(function (reply: any) {
```
Comment (hakubo) [inline front/scripts/main/models/DiscussionPostModel.ts]: It looks like map is what you want to use here

**Pull request merged**